CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+- New items in the `pull_request_template.md` ([#12](https://github.com/vtex-sites/gatsby.store/pull/12))
```

Comment:
Oooh, good call adding the link! Hadn't noticed GH didn't create the link automatically as it does inside issues and PRs messages. Will do the same moving forward.
😬

Reply:
Yes! 😃  I will this to the checklist as well!